src/app/notifications/notifications.component.html

```diff
@@ -0,0 +1,14 @@
+<div *ngFor="let notification of activeNotifications$ | async">
+  <div class="container text-center notification-bar {{ notification.priority?.toLowerCase() }}">
+    <div *ngIf="notification.type == 'SITEWIDE'">
```

Comment:
=== and there's probably an enum somewhere

Reply:
Yeah there's an enum, I have it set as an enum in my newsbody PR, so I'll fix it there.